types/src/transaction/transaction.rs

```diff
@@ -346,6 +317,109 @@ fn is_input_and_output_consistent(inputs: &[AssetTransferInput], outputs: &[Asse
     sum.iter().all(|(_, sum)| sum.is_zero())
 }
 
+fn apply_bitmask_to_output(
+    mut bitmask: Vec<u8>,
+    filter_len: usize,
```

Comment:
Why does it need `filter_len`?
Isn't it the length of `bitmask`?

Reply:
I removed that argument.